web_backend/src/main/resources/application.properties

```diff
@@ -18,6 +18,27 @@ quarkus.rest-client.leshan-event-api.http2=true
 # General
 %test.quarkus.http.test-timeout=5s
 
+quarkus.http.cors=true
+quarkus.http.cors.origins=*
+
+## OIDC Configuration
+quarkus.oidc.auth-server-url=http://localhost:4000/kc/realms/teamagochi
+quarkus.oidc.client-id=teamagochi-backend
+quarkus.oidc.credentials.secret=5DACLJH84KTWBG22UpdnS9DSjVCIu5zB
```

Comment:
Is this supposed to be public?

Reply:
Yes it is. This is a project which will never be deployed to a real server and simplicity is key.
All secrets are already public, because of [platform/data/keycloak/import](https://github.com/smartuni/teamagochi/tree/main/platform/data/keycloak/import).

Reply:
Swag all good, was just making sure